### DISCUSSION TITLE
Minor cosmetic details polishing

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#387] [Ideas]
First, I must say that the overall aesthetics improved a lot in the recent months, and is already looking very modern and polished. So, those comments are just for the sake of minor details polishing.

1- Ticked checkboxes in the menubar are just the tik.:
![image](https://user-images.githubusercontent.com/77707706/169670127-2660b408-8780-41fe-9a06-96df9c1a6b83.png)

I think it would be more visually clear if they were a tick inside a box. I mean, it´s not immediately obvious that empty checkboxes and floating ticks are the 2 sides of the same coin.
The checkboxes in the Format panel, and in the preferences window follow this suggestion:
![image](https://user-images.githubusercontent.com/77707706/169670154-df409112-b739-4134-bf3e-c86e657eb2e7.png)

2-The active version of the new icons are highres, but the innactive ones are low res. For example:
![image](https://user-images.githubusercontent.com/77707706/169670165-41763b51-3b85-43ba-b0d9-3aebcfcadbd2.png)

3-The unselected options of a list of options are blank:
![image](https://user-images.githubusercontent.com/77707706/169670217-fcae7fba-6a59-4446-84c7-1e8c63d9e18a.png)
I think that, if they were empty radio buttons, then it would be more clear that it's a list of options.
Again, the Format Panel has this idea applied:
![image](https://user-images.githubusercontent.com/77707706/169670231-401c26cb-8f78-4bdd-a389-6c1cbf42f429.png)

4-Ken Hill[ already pointed](https://sourceforge.net/p/freeplane/featurerequests/2619/) that maybe a better symbology for the styles list would be radio buttons.
![image](https://user-images.githubusercontent.com/77707706/169670294-47a4b03c-7e60-4605-af66-67f7d59e91c9.png)

5-Status line: The error notification gets this contrasting color bubble, with an (old style?) icon exceeding the borders of it:
![image](https://user-images.githubusercontent.com/77707706/169670324-7c738482-704a-40ec-a869-deef88573a3d.png)

6-Status line: I think this kind of border lines are not used in the rest of the UI, right?
![image](https://user-images.githubusercontent.com/77707706/169670349-176728ff-a6a4-4250-b0c6-9202122c60fe.png)

7-I think the splash screen is not looking modern. Maybe, a plain blue background and a sans serif font, without a shadow, would improve it. I know some picky people who would get a bad first impression right in this splash screen. And, to make ir worse, it's looking jagged on my new laptop:
![image](https://user-images.githubusercontent.com/77707706/169670365-3b5086ca-6d7d-4dc6-84df-572abd528f75.png)

8-This lost line:
![image](https://user-images.githubusercontent.com/77707706/169670399-33eddc84-04e9-411d-9d2c-614b6f00333b.png)

Again, not high priority stuff. But, it may help to conquer the heart of those who are picky about aesthetics.
BR,

**Comment by rexelbartolome** [comment]
> the innactive ones are low res.

Hmmm, is that from the toolbar? Mine are both inactive and they're pretty much the same as the 'active' ones
![image](https://user-images.githubusercontent.com/88552647/169690452-af87a566-0946-4d75-98da-687385d2768f.png)

>it's looking jagged on my new laptop

This also looks fairly okay on my setup, might be a problem with scaling on your end? Though I do agree it also needs an update.
![image](https://user-images.githubusercontent.com/88552647/169690546-1e719fcb-9afc-4744-872d-b1794053f0e7.png)

> contrasting color bubble, with an (old style?)

Yeah I think the set of icons this is pulling from also needs an update, though I have to look for what specific svg's they're referencing to. Not sure why it's overlapping the border though, might be a LaF thing that Dimitry might know more about.

> this kind of border lines are not used in the rest of the UI, right?

Yeah, I think that's also sparingly used on other parts but it doesn't look intentional, so yeah best to remove it. Though there's also another issue with that, the icon color should be a white version not a dark version. I think I know where that problem is from so I'll put it on a Github issue.

Speaking of borders, I also think this border around the 'mindmap canvas' is redundant so I think it should also be removed
![image](https://user-images.githubusercontent.com/88552647/169691300-9c6b4793-0c17-4301-b446-f27d9699ee21.png)

Everything else I pretty much agree with :)

**Reply by euu2021** [reply]
About the inactive icons and the splash screen looking jagged, I checked on my desktop, and they both look OK there. I will open a bug ticket about it.

**Reply by abc16361** [reply]
> Speaking of borders, I also think this border around the 'mindmap canvas' is redundant so I think it should also be removed

It would be great if the redundant border can be removed.

**Comment by euu2021** [comment]
About number 1 above:

I see it is changed on 1.10.3-pre5:
![image](https://user-images.githubusercontent.com/77707706/169716258-766e397d-017c-496e-8e19-29d69c379ecf.png)
![image](https://user-images.githubusercontent.com/77707706/169721917-849c2140-12b1-4bb3-aa4e-7f87519e747d.png)

In[ this thread](https://sourceforge.net/p/freeplane/discussion/758437/thread/945c81a9a3/), I learned that those checkboxes are complicated things, but I must say that, if possible, it would be nice to have them less "bold". The checked checkboxes look "bold" in comparison with the unchecked ones. Notice the big contrast between checked and unchecked:
![image](https://user-images.githubusercontent.com/77707706/169722257-166fb055-ab3c-44bf-b74d-1865137d8f9d.png)


There are, in FP itself, two other kind of checkboxes that, maybe, are better options:
![image](https://user-images.githubusercontent.com/77707706/169722190-7731a414-6bda-4913-8ebc-6f09f40ca5fd.png)
![image](https://user-images.githubusercontent.com/77707706/169722221-c4823065-648a-4a1a-a47c-a6bb515522a0.png)
